src/robots/robot-wrapper.cpp

```diff
@@ -249,14 +249,13 @@ namespace tsid
                                           const Model::FrameIndex index,
                                           Data::Matrix6x & J) const
     {
-      return se3::getFrameJacobian(m_model, data, index, J);
+      return se3::getJacobian<se3::WORLD>(m_model, data, index, J);
```

Comment:
Now we get the Jacobian in the world frame @jcarpent

Reply:
You should use m_model.frames[index].parent as the joint index to get the world jacobian of the parent joint.
se3::getJacobian<se3::WORLD>(m_model, data, m_model.frames[index].parent, J);